### DISCUSSION TITLE
Make /tpr executable by console

### DISCUSSION THREAD

**Discussion by oskarkk** [EssentialsX/Essentials#4868] [Ideas and Feature Suggestions]
It would be nice if I could /tpr someone from console. Many ways for automation with this.

**Comment by osfanbuff63** [comment]
I think this is possible via `/sudo`. I usually use `/force` instead, but that's not an EssentialsX command, and sudo should work.

So I think then `/sudo <player> tpr` then.

**Reply by oskarkk** [reply]
Does this work if the player doesn't have permission for /tpr?

**Reply by osfanbuff63** [reply]
Oh, I'm not sure. I'll look into it.

**Reply by kennyrkun** [reply]
Since sudo runs the command as a player, the command will not work if the player does not have the necessary permissions.
